DolbyIO/Source/Private/DolbyIOVideoSink.h

```diff
@@ -25,14 +27,12 @@ namespace DolbyIO
 		void handle_frame(std::unique_ptr<dolbyio::comms::video_frame>) override;
 
 		void RecreateIfNeeded(int Width, int Height);
-		void Convert(dolbyio::comms::video_frame&);
+		void Convert(dolbyio::comms::video_frame& VideoFrame);
 
 		void UpdateMaterial(UMaterialInstanceDynamic* Material);
 
 		UTexture2D* Texture{};
-		TArray<uint8> Buffer;
-		FUpdateTextureRegion2D Region{0, 0, 0, 0, 0, 0};
-		FRenderCommandFence Fence;
+		// FRenderCommandFence Fence;
```

Comment:
If this isn't needed anymore, then the RHI header can possibly be removed as well.